map_test.go

```diff
@@ -138,6 +138,32 @@ func TestMatchEmptyOrDeleted(t *testing.T) {
 	}
 }
 
+func TestConvertDeletedToEmptyAndFullToDeleted(t *testing.T) {
+	ctrls := make([]ctrl, groupSize)
+	expected := make([]ctrl, groupSize)
+	for i := 0; i < 100; i++ {
+		for j := 0; j < groupSize; j++ {
+			switch rand.Intn(4) {
+			case 0: // 25% empty
+				ctrls[j] = ctrlEmpty
+				expected[j] = ctrlEmpty
+			case 1: // 25% deleted
+				ctrls[j] = ctrlDeleted
+				expected[j] = ctrlEmpty
+			case 2: // 25% sentinel
+				ctrls[j] = ctrlDeleted
```

Comment:
These should be `ctrlSentinel` and `ctrlEmpty`

Reply:
I can fix this.

Reply:
Oops. Please do.